src/plugins/gui/amountQuotePlugin.ts

```diff
@@ -107,12 +107,12 @@ export const amountQuoteFiatPlugin: FiatPluginFactory = async (params: FiatPlugi
       // Pop up modal to pick wallet/asset
       // TODO: Filter wallets according to fiats supported by allowed providers
```

Comment:
This TODO seems outdated now. Looks like we're just going to always force a fiat
if required, regardless of the wallet fiat

Reply:
Not always. There are cases where a payment method can allow different fiats. ie Visa/MC